pysatModels/utils/extract.py

```diff
@@ -627,10 +647,24 @@ def extract_modelled_observations(inst, model, inst_name, mod_name,
         raise ValueError("".join(["unknown model name for datetime: ",
                                   mod_datetime_name]))
 
-    tm_sec = (mod_datetime[1:] - mod_datetime[:-1]).min()
-    tm_sec /= np.timedelta64(1, 's')
-    ti_sec = (inst.index[1:] - inst.index[:-1]).min().total_seconds()
-    min_del = tm_sec if tm_sec < ti_sec else ti_sec
+    # Determine the appropriate time difference in seconds from the instrument
+    # and model data.  If only one time value is present, assume anything is
+    # close enough
+    dtime = mod_datetime[1:] - mod_datetime[:-1]
```

Comment:
Similar comment here as given below.

Reply:
Not needed here, since these were assigned using values